api/httpjson/RPCserver.go

```diff
@@ -109,6 +109,19 @@ func (s *RPCServer) Handle(w http.ResponseWriter, r *http.Request) {
 			w.Write(data)
 			return
 		}
+		// if params["RemoteAddr"] set but empty, used request.RemoteAddr
+		if addr, ok := params["RemoteAddr"]; ok {
```

Comment:
Why not set RemoteAddr it if params["RemoteAddr"] is not set?

Reply:
params["RemoteAddr"] just only significant for getid4extip RPC. It is condition for distinguish getid4extip or others RPC during Handle() routine.

Reply:
OK